R/rc_helpers.R

```diff
@@ -33,7 +33,12 @@ cf_ind_importance <- function(x) {
 #' @export
 #'
 rc_update_weather <- function(dt = NULL){
+  # Add visible bindings
+  W_POT_TO_ACT = NULL
```

Comment:
This naming is confusing. I would suggest something like W_ET_FACTOR_POTACT or something similiar. @BrentWHR what would you suggest?